index.ts

```diff
@@ -161,19 +177,28 @@ export default function install(app: App, {
 }
 
 export function useTranslation() {
-    const instance = getCurrentInstance();
-    if (!instance) {
-        throw new Error("i18next-vue: No Vue instance in context. Make sure to register the i18next-vue plugin using app.use(...).");
-    }
+    const instance = currentInstance();
     const globalProps = instance.appContext.config.globalProperties;
     return {
         i18next: globalProps.$i18next as i18n,
         t: globalProps.$t.bind(instance.proxy) as SimpleTFunction
     }
 }
 
+function currentInstance() {
+    const instance = getCurrentInstance();
+    if (!instance) {
+        throw new Error("i18next-vue: No Vue instance in context. Make sure to register the i18next-vue plugin using app.use(...).");
+    }
+    return instance;
+}
+
 // pattern matches '{ someSlot }'
-const slotNamePattern = new RegExp('{\\s*([a-z0-9\\-]+)\\s*}', 'gi');
+function slotNamePattern({start = '{', end = '}'}) {
```

Comment:
Did you have a particular reason to use a parameter object instead of "regular" parameters?

Otherwise `slotNamePattern(start = '{', end = '}')` should also work.